### DISCUSSION TITLE
signAndSendAllTransactions equivalent for VersionedTransactions?

### DISCUSSION THREAD

**Discussion by adrian-petrykiw** [phantom/docs#288] [Help & Support] [answered]
### Summary

hey all with the switch to VersionedTransactions is there some equivalent to the signAndSendAllTransactions that would be used with legacy transactions? or is this not possible anymore bc of how VersionedTransactions work?

### Example

https://docs.phantom.app/solana/sending-a-transaction#signing-multiple-transactions

### Steps to Reproduce

versioned transactions according to phantom docs are built like so:

const provider = getProvider(); // see "Detecting the Provider"
const network = "<NETWORK_URL>";
const connection = new Connection(network);
const versionedTransaction = new VersionedTransaction();
const { signature } = await provider.signAndSendTransaction(versionedTransaction);
await connection.getSignatureStatus(signature);

passing an array of VersionedTransactions in a method like signAndSendAllTransactions does not seem to be possible?

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@adrian-petrykiw Sorry for the delay in response! signAndSendAll should be usable w/ versioned transactions. Please let me know if you see other wise and provide any relevant code.
